src/PieChart.js

```diff
@@ -141,10 +147,11 @@ class PieChart extends React.Component {
     onMouseLeaveSlice: PropTypes.func,
   };
   static defaultProps = {
-    getValue: null,
+    slice: null,
```

Comment:
Please remove slice as a defaultProp since we're explicitly marking it as required for the user to pass into the component API.